### PR TITLE
Pull Request Julian 

### PR DESCRIPTION
**Pull request by hulianbrox**

### CODE REVIEW

File: motion/p5js/motion-without-motion/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+  function setup() {
+      createCanvas(400,400);
```
Comment (fabianmoronzirfas) [inline motion/p5js/motion-without-motion/index.js]: The canvas should be 500 by 500 and assigned to the sketch id div. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code

File: motion/p5js/motion-without-motion/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      noStroke();
+      colorMode(HSB, 360, 100, 100, 100);
+      background(49,63,100);
+      fill (348,63,100,20); 
+      
+      myEllipse (50,250);
```
Comment (fabianmoronzirfas) [inline motion/p5js/motion-without-motion/index.js]: So many magic numbers…
Changing the size will break these.

File: motion/p5js/motion-without-motion/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      ellipse (a,b,50,50);
+      a +=12.5;
+      b -=12.5;
+      }
+      } // END SETUP 
+      
+      function draw() {
+      }
+      
+      function mousePressed(){
```
Comment (fabianmoronzirfas) [inline motion/p5js/motion-without-motion/index.js]: Please remove the image saving code.

File: motion/p5js/raindrops/index.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+var d1 = 8;
+
+//Rectangle 
+
+var rectX;
+var rectH;
+
+
+function setup() {
+   createCanvas(500,500);
```
Comment (fabianmoronzirfas) [inline motion/p5js/raindrops/index.js]: assigned to the sketch id div https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code

File: motion/p5js/raindrops/index.js [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+}
+  
+  //Line  
+  stroke(227,68,40);
+  rect(0, rectX, width, rectH);
+
+ // Klammer End Draw 
+}
+
+function mousePressed () {
```
Comment (fabianmoronzirfas) [inline motion/p5js/raindrops/index.js]: remove the image saving.

File: motion/p5js/raindrops/index.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---  
+layout: p5js
+author: Julian Broocks
+author-url: https://github.com/hulianbrox
+title: raindrops
+tags: 
+  - P5.js
+  - motion
+  - law of similarity
```
Comment (fabianmoronzirfas) [inline motion/p5js/raindrops/index.md]: it should be 

    Law of Similarity

File: motion/p5js/raindrops/index.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---  
+layout: p5js
+author: Julian Broocks
+author-url: https://github.com/hulianbrox
+title: raindrops
```
Comment (fabianmoronzirfas) [inline motion/p5js/raindrops/index.md]: Title case please.

File: projects/gedankenblitze/index.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+  - "Julian Broocks"
+authors-url: 
+  - "https://github.com/hulianbrox"
+layout: project
+title: "Gedankenblitze"
+type: project
+---
+![Gedankenblitze](splash.png)
+
+<h2>The assignment</h2>
```
Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: Please use markdown syntax not HTML

File: projects/gedankenblitze/index.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+The assignment was to create an algorithm that had to be sketched on paper by 10 test persons. Content of the algorithm was a simple drawing instruction after defining parameters like tools, format, time and material. The description should be precise and easy to execute while leaving room for free interpretation. Further on, more iterations were conducted by changing parameters of the algorithm to improve its description and output. 
+
+„Analog algorithm“ was the first assignment of »Designing with code – the basic of generative design« by Fabian Morón Zirfas at the University of Applied Sciences Potsdam.
+
+<h2>My inspiration</h2>
```
Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: markdown not HTML (I wont comment on the rest.)

File: projects/gedankenblitze/index.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+I was inspired by the shapes of lightning which can be recreated with paper by crumpling it. Furthermore, I drew inspiration from an artwork by Anders Hoff, who published a study of generative algorithms. The centralized arrangement of the elements in the picture really appealed to me. So I wanted to implement this aspect into my algorithm.
```
Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: Please write from an objective perspective. Loose the I, we, my…

File: projects/gedankenblitze/index.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+<h2>The assignment</h2>
+
+The assignment was to create an algorithm that had to be sketched on paper by 10 test persons. Content of the algorithm was a simple drawing instruction after defining parameters like tools, format, time and material. The description should be precise and easy to execute while leaving room for free interpretation. Further on, more iterations were conducted by changing parameters of the algorithm to improve its description and output. 
+
+„Analog algorithm“ was the first assignment of »Designing with code – the basic of generative design« by Fabian Morón Zirfas at the University of Applied Sciences Potsdam.
+
+<h2>My inspiration</h2>
+I was inspired by the shapes of lightning which can be recreated with paper by crumpling it. Furthermore, I drew inspiration from an artwork by Anders Hoff, who published a study of generative algorithms. The centralized arrangement of the elements in the picture really appealed to me. So I wanted to implement this aspect into my algorithm. 
+
+![Inspiration](assets/inspiration.png) </br>
```
Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: paths need to be relative

File: projects/gedankenblitze/index.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+(source http://inconvergent.net/)
```
Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: URLs as links please.

File: projects/gedankenblitze/index.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+a white piece of paper: »background layer«</br>
+a tracing paper: »drawing layer«</br>
+tools: 3 pens of different thickness
+
+
+<h3>Background layer</h3>
+Thick pen: Draw an oval or circle which is almost as big as the drawing surface. This is your auxiliary line. Divide this line in 2 - 4 parts. There are gaps between these parts which are as wide as 2 - 5 fingers. 
+
+<h3>On the tracing paper</h3>
+Crumple the paper and flatten it again. Put it on top of the background layer.
```
Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: The algorithm could be displayed as quote:  

> Now trace the rigid lines which came from crumpling the paper.  
> Thick Pen: Trace the rigid lines along the auxiliary line.   
> Medium-size pen: Draw one to three branches from each line into the centre of the oval/circle.  
> Thin pen: Repeat the previous step one to two times.   

This is done like this:  

    > Now trace the rigid lines which came from crumpling the paper.  
    > Thick Pen: Trace the rigid lines along the auxiliary line.   
    > Medium-size pen: Draw one to three branches from each line into the centre of the oval/circle.  
    > Thin pen: Repeat the previous step one to two times.

File: projects/lyonel_feininger/index.md [hunk 14]
```diff
@@ -0,0 +1,166 @@
...
+  - "https://github.com/hulianbrox"
+  - "https://github.com/jakobvk"
+layout: project
+title: "Lyonel Feininger"
+type: project
+---
+
+![Vogelwolke](splash.png)
+
+<h1> Deconstructing Master Artists </h1>
```
Comment (fabianmoronzirfas) [inline projects/lyonel_feininger/index.md]: Not HTML - MD please

Comment (fabianmoronzirfas) [inline projects/lyonel_feininger/index.md]: Also the title is added by the system - see https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax

**Comment by hulianbrox** [comment]
How can I create the subfolder for the images?

**Comment by fabianmoronzirfas** [comment]
> How can I create the subfolder for the images?  
Are you using the GitHub app? You should. You can just create a folder and commit it. 
Did you google for an answer? This is the second result I got for the search term "github web interface move file" --> https://github.com/blog/1436-moving-and-renaming-files-on-github

**Comment by fabianmoronzirfas** [comment]
Can I review this again?

**Comment by fabianmoronzirfas** [comment]
Also do you have the rights to redistribute these images?

**Comment by fabianmoronzirfas** [comment]
It's really annoying that you don't use the GitHub app. I get an email for each time you do a commit

File: projects/gedankenblitze/index.md [hunk 15]
```diff
@@ -18,7 +18,7 @@ The assignment was to create an algorithm that had to be sketched on paper by 10
 ## My inspiration 
 Inspiration came from the shapes of lightning, which can be recreated with paper by crumpling it, and from an artwork by Anders Hoff, who published a study of generative algorithms. The centralized arrangement of the elements in the picture were really appealing. So this aspect was implemented into the algorithm. 
 
-![Inspiration](assets/images/inspiration.png) </br>
+![Inspiration](../inspiration.png) </br>
```
Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: This is wrong. `../` goes one folder done. So your images would be lying around in the projects folder?

**Comment by hulianbrox** [comment]
ok I did the changes

File: projects/lyonel_feininger/index.md [hunk 16]
```diff
@@ -22,20 +22,20 @@ The project was developed for the class “Gestalten in Code – Die Grundlagen
 Due to his works at Bauhaus 1919, Lyonel Feininger is one of the most eminent artist of the period of the “Classical Modernity”. 
 Lyonel Feininger was born 1871 in New York City. When he came to Europe 1887 he studied in Hamburg, Berlin and then Paris, France. 1919 he was called to Bauhaus University by Walter Gropius to be the first school master for painting and graphics. 
 
-![Lyonel-Feininger](assets/001_lyonel_feininger.png)
+![Lyonel-Feininger](assets/images/001_lyonel_feininger.png)
```
Comment (fabianmoronzirfas) [inline projects/lyonel_feininger/index.md]: the paths to the files are still not relativ it has to be   

      

the `./` before assets is important

File: projects/lyonel_feininger/index.md [hunk 17]
```diff
@@ -22,20 +22,20 @@ The project was developed for the class “Gestalten in Code – Die Grundlagen
...
 
 ## “Vogelwolke” 1924
 
 The painting “Vogelwolke” from 1924 was drawn in the middle of Lyonel Feiningers productive years.
 
-![Vogelwolke](assets/004_vogelwolke.png) 
+![Vogelwolke](assets/images/004_vogelwolke.png) 
 
 ##Why did we pick it?
 When looking for an artist and pieces of art for this project, we enjoyed the style of Lyonel Feiningers paintings because they, on the one hand side, appear to be only abstract shapes, but on the other hand still portray nature.
```
Comment (fabianmoronzirfas) [inline projects/lyonel_feininger/index.md]: Please try to write from an objective perspective. No we, I, my! Can you describe your project without using the first person?

Comment (hulianbrox) [inline projects/lyonel_feininger/index.md]: But this is the paragraph about our personal reasons, why we chose this artpiece, to us it only makes sense to write this also from a personal perspective.

**Comment by fabianmoronzirfas** [comment]
Okay. Does this count as ready for review?

**Comment by hulianbrox** [comment]
Yes

**Comment by fabianmoronzirfas** [comment]
@hulianbrox What about the rights for the fenyinger images?

**Comment by hulianbrox** [comment]
We can't find any images of "Vogelwolke" with a label to distribute the image. What should we do?

**Comment by fabianmoronzirfas** [comment]
Did you try a google search for the actual image?

**Comment by fabianmoronzirfas** [comment]
https://support.google.com/websearch/answer/1325808?hl=de

**Comment by hulianbrox** [comment]
Still can't find one

**Comment by hulianbrox** [comment]
Ok I am done ;) deleted the images

**Comment by fabianmoronzirfas** [comment]
Oki.

**Comment by fabianmoronzirfas** [comment]
@hulianbrox nope the images are still there.

**Comment by hulianbrox** [comment]
Oh, just noticed I forgot the images in the assets folder. Deleted those too now.

**Comment by fabianmoronzirfas** [comment]
So do you have no no images at all for the project?

**Comment by hulianbrox** [comment]
I added a link for people to see the picture 'Vogelwolke'. There is no image on the web with the right to distribute it. Same thing for the portrait of Lyonel Feininger. We do have pictures and these are the important ones: the images visualising our analysis of the composition and Gestalt Laws.

**Comment by fabianmoronzirfas** [comment]
Did you take a look at this? http://www.harvardartmuseums.org/collections/object/304376?q=Vogelw You can write a request to use the image. If you tell them that you did an anlysis of it and we will put it up on a website you might get the right to use a low resolution image.

**Comment by fabianmoronzirfas** [comment]
Also the portrait on this page https://de.wikipedia.org/wiki/Lyonel_Feininger looks like there is also no copy right on it

**Comment by fabianmoronzirfas** [comment]
https://de.wikipedia.org/wiki/Lyonel_Feininger#/media/File:Fuenfundneunzig_Koepfe_von_Orlik_-,_Mit_einem_Vorwort_von_Max_Osborn_(3586405810).jpg

**Comment by hulianbrox** [comment]
Saw that illustration aswell, but didn't think it was suitable. But if you think so I will add it. Thanks for the link to the harvart museum, that could be a good solution so I contacted them.

**Comment by hulianbrox** [comment]
Ok that's it for tonight haha The Havard Art Museum even replied and granted the right to publish the image. I am waiting for Anders Hoff's answer, other than that I hope I am good to go now.

**Comment by fabianmoronzirfas** [comment]
Awesome

**Comment by hulianbrox** [comment]
Anders Hoff hasn't answered until now. Would you accept my documentation without this picture?

**Comment by fabianmoronzirfas** [comment]
How did you try to contact him? On Twitter? Per E-Mail? Here on GitHub? Did you search his site for some copyright notice?

**Comment by hulianbrox** [comment]
Emailed him and tried to reach him via Instagram, no luck. There is an MIT license on Github for the software but not for the artworks. On the official website he is selling some of the art pieces for 300 bucks and it only says "Copyright Anders Hoff".

**Comment by fabianmoronzirfas** [comment]
He seems to be pretty active on Twitter. You might try this one. Which are the images from him? Can you point me to them and give me the link where you got them?

**Comment by hulianbrox** [comment]
http://inconvergent.net/generative/differential-mesh-3d/ It's just the
first one. Created an account on twitter now and contacted him.

2017-06-09 7:23 GMT+02:00 Fabian Morón Zirfas <notifications@github.com>:

**Comment by fabianmoronzirfas** [comment]
Take a look at the linked [repository](https://github.com/inconvergent/differential-mesh-3d). The repo is under MIT License.

**Comment by hulianbrox** [comment]
The MIT license is only referring to the software and not explicitly to the art pieces. But he actually replied on Twitter :) Could you accept the pull request then? I did all the changes you asked for.

**Comment by fabianmoronzirfas** [comment]
Cool. I'll review it tomorrow.

File: motion/p5js/motion-without-motion/index.js (hunk 1 above)
Comment (fabianmoronzirfas) [inline motion/p5js/motion-without-motion/index.js]: The canvas should be 500 by 500 and assigned to the sketch id div. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code

File: motion/p5js/motion-without-motion/index.js [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+      background(49,63,100);
+      fill (348,63,100,20); 
+      
+      myEllipse (50,250);
+      myEllipse (162.5,375);
+      myEllipse (350,150);
+      
+      // - - - - - - Function - - - - - - 
+      function myEllipse (xe, ye) {
+      push();
```
Comment (fabianmoronzirfas) [inline motion/p5js/motion-without-motion/index.js]: Does this actually have an effect? The `push` `pop` commands need to be called in context with `translate` or `rotate`. Please check it.

File: motion/p5js/motion-without-motion/index.js [hunk 19]
```diff
@@ -0,0 +1,55 @@
...
+      a +=12.5;
+      b -=12.5;
+      }
+      } // END SETUP 
+      
+      function draw() {
+      }
+      
+      function mousePressed(){
+      save('image.jpg');
```
Comment (fabianmoronzirfas) [inline motion/p5js/motion-without-motion/index.js]: Please remove the image saving code

File: motion/p5js/motion-without-motion/index.md [hunk 20]
```diff
@@ -0,0 +1,12 @@
+---  
+layout: p5js
+author: Julian Broocks
+author-url: https://github.com/hulianbrox
+title: motion without motion
```
Comment (fabianmoronzirfas) [inline motion/p5js/motion-without-motion/index.md]: Title Case please

File: motion/p5js/motion-without-motion/index.md [hunk 21]
```diff
@@ -0,0 +1,12 @@
...
+tags: 
+  - P5.js
+  - motion
+  - repeat
+permalink: /motion/motion-without-motion/
+summary: How to create motion without moving elements.
```
Comment (fabianmoronzirfas) [inline motion/p5js/motion-without-motion/index.md]: Is this a question?

File: motion/p5js/raindrops/index.md [hunk 22]
```diff
@@ -0,0 +1,12 @@
...
+layout: p5js
+author: Julian Broocks
+author-url: https://github.com/hulianbrox
+title: Raindrops
+tags: 
+  - P5.js
+  - motion
+  - Law of Similarity
+permalink: /motion/raindrops/
+summary: Elements of same speed towards same direction are seen as grouped together.
```
Comment (fabianmoronzirfas) [inline motion/p5js/raindrops/index.md]: Please make this a real sentence:  

 Elements of same speed towards **the** same direction are seen as a group ~~grouped together~~.

File: projects/gedankenblitze/index.md [hunk 23]
```diff
@@ -0,0 +1,72 @@
...
+
+## My Inspiration 
+Inspiration came from the shapes of lightning, which can be recreated with paper by crumpling it, and from an artwork by Anders Hoff, who published a study of generative algorithms. The centralized arrangement of the elements in the picture were really appealing. So this aspect was implemented into the algorithm. 
+
+![Differnitalmesh3D](./assets/images/inconvergent.png)
+copyrighted by [http://inconvergent.net](http://inconvergent.net/) 
+
+## The Algorithm 
+
+> ### Material and Tools
```
Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: I'm not sure if the h3 (###) works in quotes.

File: projects/lyonel_feininger/assets/images/test.js [hunk 24]
```diff
@@ -0,0 +1 @@
+hallo
```
Comment (fabianmoronzirfas) [inline projects/lyonel_feininger/assets/images/test.js]: please remove this file. Seems just like a test

File: projects/gedankenblitze/index.md (hunk 23 above)
Comment (hulianbrox) [inline projects/gedankenblitze/index.md]: looks like it is working 
https://daringfireball.net/projects/markdown/syntax

Comment (fabianmoronzirfas) [inline projects/gedankenblitze/index.md]: Yeah we will see. Jekyll does not use the original markdown renderer from Gruber

**Comment by fabianmoronzirfas** [comment]
Thanks for the contribution. Merged an live  

- https://interface.fh-potsdam.de/gestalten-in-code/projects/gedankenblitze/
- https://interface.fh-potsdam.de/gestalten-in-code/motion/motion-without-motion/
- https://interface.fh-potsdam.de/gestalten-in-code/motion/raindrops/
- https://interface.fh-potsdam.de/gestalten-in-code/projects/lyonel_feininger/

_(even though I still had to fix some errors)_

**Pull request merged**